CONTRIBUTING.md

```diff
@@ -1,39 +1,81 @@
 
 This project adheres to the [Contributor Covenant Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code.
 
-There are a few ways you can contribute: improving an existing topic page, or curating a new topic page.
+There are a few ways you can contribute:
+
+- Improving an existing topic or collection
+- Curating a new topic topic or collection
 
 As you write content, check out the [Style Guide](./docs/styleguide.md) to learn what each field means, and how they should be formatted. Following the style guide will improve the chances of your contribution being accepted.
 
-Note: updates won't immediately appear once we've merged your PR. We pull in these changes regularly to GitHub.
+Note: Updates won't immediately appear once we've merged your PR. We pull in these changes regularly to GitHub.
 
-## Improving an existing topic page
+## Improving an existing topic or collection
 
-If a topic page already exists, it will be listed in the [topics directory](https://github.com/github/explore/tree/master/topics). The topic name should match its URL, e.g. `https://github.com/topics/rails` corresponds to the [`rails` folder](https://github.com/github/explore/tree/master/topics/rails).
+If a topic or collection already exists, it will be listed in its respective directory:
 
-To improve an existing topic page, please **open a pull request** with your proposed changes:
+- [topics/](https://github.com/github/explore/tree/master/topics)
+- [collections/](https://github.com/github/explore/tree/master/collections)
 
-* To update the image, replace the image inside the topic's folder
-* To update the content, edit the `index.md` inside the topic's folder
+The topic or collection name should match its URL, e.g. `https://github.com/topics/rails` corresponds to the [`topics/rails` directory](https://github.com/github/explore/tree/master/topics/rails).
 
-The [pull request template](./.github/PULL_REQUEST_TEMPLATE.md) provides guidance on the information you need to include.
+To make an improvement, please **open a pull request** with your proposed changes:
 
-**Please fill out the pull request template completely.** If you do not fill out the template, your PR will be closed.
+### Update the image
+
+To update the image, simply replace the image inside the topic or collection's directory.
+
+### Update text and links
+
+To update text and links, edit the `index.md` inside the topic or collection's directory. These files are formatted using a combination of [Front Matter](https://jekyllrb.com/docs/frontmatter/) and simple body content.
+
+For **topics**, you'll notice that, in examples like the topic "[algorithm](https://raw.githubusercontent.com/github/explore/master/topics/algorithm/index.md)," data like the its canonical URL, Wikipedia URL, or display name are called out in key-value pairs; while its detailed description is accounted for in the body of the document.
```

Comment:
> like the its canonical URL

I think you want to drop either 'the' or 'its', but also, the algorithm example doesn't have a `url` field.